manuscript/090_How_to_start.md

```diff
@@ -23,27 +23,27 @@ As I
 and try for yourself how difficult it is to answer the following questions:
 
 1.  How do you know what situation each Statement describes?
-2.  How do you know whether the Statement describes a�single situation, or few of them at the same time?
-3.  How do you know whether the assertions inside those Statements are really the right ones assuming each Statement was written by someone else or a�long time ago?
+2.  How do you know whether the Statement describes a single situation, or several at the same time?
+3.  How do you know whether the assertions inside those Statements are really the right ones assuming each Statement was written by someone else or a long time ago?
 4.  How do you know whether the Statement should stay or be removed when you modify the functionality it specifies?
-5.  If your changes in production code make a�Statement evaluate to false, how do you know whether the Statement is no longer correct or the production code is wrong?
-6.  How do you know whether you will not introduce a�duplicate Statement for a�behavior that is already specified by another Statement when adding to Specification originally created by another team member?
-7.  How do you estimate, by looking at the runner tool report, whether the fix for failing Statement will be easy or not?
-8.  How do you answer a�new developer in your team when they ask you �what is this Statement for?"
-9.  How can you keep track of the Statements already made about the specified class vs those that still need to be made?
+5.  If your changes in production code make a Statement evaluate to false, how do you know whether the Statement is no longer correct or the production code is wrong?
+6.  How do you know whether you will not introduce a duplicate Statement for a behavior when adding to a Specification that was originally created by another team member?
+7.  How do you estimate, by looking at the runner tool report, whether the fix for a failing Statement will be easy or not?
+8.  What do you answer new developers in your team when they ask you “what is this Statement for?"
+9.  How can you keep track of the Statements already made about the specified class and those still to make?
```

Comment:
Saying "Statements already made about the specified class and those still to make" makes it look like these may be two separate things to keep track of. What I mean when I said "Statements already made about the specified class vs those that still need to be made" was a single thing - i.e. the specification completeness. 

The point is that when we have names like Test01, Test02, Test03 there's no chance telling how complete the specification is. Also, it's very hard to determine what is missing, e.g. when there are 7 behaviors and we have the three Statements, which ones are already covered? The first three? The last three?

Is there a good way to write this? I don't mind if it's a bit more verbose, what's important to me is getting the point across.